reference/render/list-button--button-selected.html

```diff
@@ -0,0 +1,6 @@
+<button type="button" class="btn btn-list-button list-button-selected">
```

Comment:
Nu kan hooguit visueel worden bepaald dat iets geselecteerd is. Volgens mij moet daar ofwel een aria- attribuut bij (`aria-pressed`), en/of in de buttontekst moet duidelijk worden dat een button geselecteerd is.

Reply:
Ik heb een `<span class="sr-only">geselecteerd</span>` toegevoegd. `aria-pressed` vind ik een beetje raar in de context van '99x geselecteerd'?